tox.ini

```diff
@@ -1,5 +1,6 @@
 [tox]
 envlist = packaging, pep8, py35, py36, py37, py38
+isolated_build = True
```

Comment:
For onlookers who didn't know what this meant (me!):

> Activate isolated build environment. tox will use a virtual environment to build a source distribution from the source tree. For build tools and arguments use the pyproject.toml file as specified in PEP-517 and PEP-518. To specify the virtual environment Python version define use the isolated_build_env config section.

— [tox](https://tox.wiki/en/latest/config.html#conf-isolated_build)